modules/FindImGui.cmake

```diff
@@ -0,0 +1,93 @@
+#.rst:
+# Find ImGui
+# -------------
+#
+# Finds the ImGui library. This module defines:
+#
+#  ImGui_FOUND                - True if ImGui library is found
+#  ImGui::ImGui               - ImGui imported target (only on CMake >= 3.0,
+#   because interface targets are not supported on 2.8.12)
+#
+# Additionally these variables are defined for internal usage:
+#
+#  ImGui_INCLUDE_DIR          - Include dir
+#
+
+#
+#   This file is part of Magnum.
+#
+#   Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018
+#             Vladimír Vondruš <mosra@centrum.cz>
+#   Copyright © 2018 Jonathan Hale <squareys@googlemail.com>
+#
+#   Permission is hereby granted, free of charge, to any person obtaining a
+#   copy of this software and associated documentation files (the "Software"),
+#   to deal in the Software without restriction, including without limitation
+#   the rights to use, copy, modify, merge, publish, distribute, sublicense,
+#   and/or sell copies of the Software, and to permit persons to whom the
+#   Software is furnished to do so, subject to the following conditions:
+#
+#   The above copyright notice and this permission notice shall be included
+#   in all copies or substantial portions of the Software.
+#
+#   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+#   THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#   DEALINGS IN THE SOFTWARE.
+#
+
+# vcpkg installs cmake package config files for imgui. We're making an INTERFACE
+# target for it, which is supported only since CMake 3.0.
+if(NOT CMAKE_VERSION VERSION_LESS 3.0.0)
```

Comment:
Just assume CMake 3.0 is used, I'll be dropping 2.8.12 soon (right now in the process of removing GCC 4.7 support, this comes right after).